### PR TITLE
[ZEPPELIN-6038] Unification of the Logger variable

### PR DESCRIPTION
**Pull request by ParkGyeongTae**
### What is this PR for?
I discovered that the Logger variable differs across files. I want to standardize the Logger variable as "LOGGER" in all files. Please let me know if there are any additional parts that need to be addressed.

### What type of PR is it?
Improvement

### Todos
* [x] - log -> LOGGER
* [x] - LOG -> LOGGER
* [x] - logger -> LOGGER
* [x] - _LOGGER -> LOGGER

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-6038

### How should this be tested?
* CI
* Build and run the Zeppelin project

### Screenshots (if appropriate)
240730_Logger

### Questions:
* Does the license files need to update? No.
* Is there breaking changes for older versions? No.
* Does this needs documentation? No.

### CODE REVIEW

**Comment by Reamer** [comment]
Please ensure that the logger is also “private static final”.

**Comment by ParkGyeongTae** [comment]
@Reamer I have set all logger constants to `private static final`. Please review this.

File: file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java [hunk 1]
```diff
@@ -33,11 +33,14 @@
...
 import org.apache.zeppelin.interpreter.InterpreterException;
 import org.apache.zeppelin.interpreter.thrift.InterpreterCompletion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * HDFS implementation of File interpreter for Zeppelin.
  */
 public class HDFSFileInterpreter extends FileInterpreter {
+  private static final Logger LOGGER = LoggerFactory.getLogger(HDFSFileInterpreter.class);
```
Comment (Reamer) [inline file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java:43]: :+1: 
It is good that these changes are included with this unification.

File: file/src/test/java/org/apache/zeppelin/file/HDFSFileInterpreterTest.java [hunk 2]
```diff
@@ -34,6 +34,7 @@
 import org.apache.zeppelin.interpreter.InterpreterResult;
 import org.apache.zeppelin.interpreter.thrift.InterpreterCompletion;
 import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
```
Comment (Reamer) [inline file/src/test/java/org/apache/zeppelin/file/HDFSFileInterpreterTest.java:37]: Please sort the imports here. I would delete `import org.slf4j.logger;` in line `26` and add it here.

File: python/src/test/java/org/apache/zeppelin/python/PythonInterpreterPandasSqlTest.java [hunk 3]
```diff
@@ -58,7 +58,7 @@
 abstract class PythonInterpreterPandasSqlTest {
 
   private static final Logger LOGGER =
-      LoggerFactory.getLogger(PythonInterpreterPandasSqlTest.class);
+          LoggerFactory.getLogger(PythonInterpreterPandasSqlTest.class);
```
Comment (Reamer) [inline python/src/test/java/org/apache/zeppelin/python/PythonInterpreterPandasSqlTest.java:61]: Unnecessary change.

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java [hunk 4]
```diff
@@ -1206,20 +1206,20 @@ private void patchParagraph(NotebookSocket conn,
     if (!zConf.isZeppelinNotebookCollaborativeModeEnable()) {
       return;
     }
-    String paragraphId = fromMessage.getType("id", LOG);
+    String paragraphId = fromMessage.getType("id", LOGGER);
```
Comment (Reamer) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java:1209]: These lines really made me wonder. Passing the logger into a function so that another class can log via it...
However, your change at this point is correct.

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java [hunk 5]
```diff
@@ -1605,7 +1605,7 @@ private void runAllParagraphs(NotebookSocket conn,
             });
         }
       } catch (Throwable t) {
-        NotebookServer.LOG.error("Error in running all paragraphs", t);
+        NotebookServer.LOGGER.error("Error in running all paragraphs", t);
```
Comment (Reamer) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java:1608]:
```suggestion
        LOGGER.error("Error in running all paragraphs", t);
```

**Review by Reamer — CHANGES_REQUESTED** [review]
Found some minor issues and a major issue.
Files within `zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/thrift` must not be edited. These files are automatically generated.

File: file/src/test/java/org/apache/zeppelin/file/HDFSFileInterpreterTest.java (hunk 2 above)
Comment (ParkGyeongTae) [reply file/src/test/java/org/apache/zeppelin/file/HDFSFileInterpreterTest.java]: I have completed this task.

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java (hunk 5 above)
Comment (ParkGyeongTae) [reply zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: I have completed this task.

File: python/src/test/java/org/apache/zeppelin/python/PythonInterpreterPandasSqlTest.java (hunk 3 above)
Comment (ParkGyeongTae) [reply python/src/test/java/org/apache/zeppelin/python/PythonInterpreterPandasSqlTest.java]: I have completed this task.

File: file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java (hunk 1 above)
Comment (ParkGyeongTae) [reply file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java:43]: :)

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java (hunk 4 above)
Comment (ParkGyeongTae) [reply zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java:1209]: :)

**Review by Reamer — APPROVED** [review]
LGTM
